### DISCUSSION TITLE
Change the timeseries value before simulation

### DISCUSSION THREAD

**Discussion by ruilee16** [pyswmm/pyswmm#385] [Q&A]
Hi, is there a way to change the timeseries values before the simulation starts?

**Comment by bemcdonnell** [comment]
@ruilee16, The best way to do this might be for you to pre-process the INP file to use a timeseries file. You would just save the timeseries file and ask SWMM to use it.  Depending on what kind of timeseries you are working with, if it is a rainfall, inflow, or boundary condition, you can just set those values during the running simulation rather than pre-process a timeseries file.
